ServerCore/Pages/Puzzles/Edit.cshtml.cs

```diff
@@ -130,6 +130,9 @@ public async Task<IActionResult> OnPostAsync(int puzzleId)
                                                       join PuzzleStatePerTeam pspt in _context.PuzzleStatePerTeam on tm.Team equals pspt.Team
                                                       where pspt.PuzzleID == Puzzle.ID && pspt.UnlockedTime != null
                                                       select tm.Member.Email).ToListAsync();
+                    List<Team> teams = await (from PuzzleStatePerTeam pspt in _context.PuzzleStatePerTeam
+                                                      where pspt.PuzzleID == Puzzle.ID && pspt.UnlockedTime != null
```

Comment:
I am following the practice of sending a notification whenever we send mail. We send mail in this case because we wanted a team that has solved a puzzle yet reported an issue to see that their reported issue was addressed.